docs/package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "docs",
   "private": true,
+  "type": "module",
```

Comment:
This is important to fix a build issue:
![Uploading CleanShot 2023-09-20 at 14.47.54@2x.png…]()